master/internal/db/postgres_test_utils.go

```diff
@@ -47,8 +47,6 @@ const (
 	DefaultTestSrcPath = "../../../examples/tutorials/mnist_pytorch"
 	// DefaultProjectID is the project used when none is specified.
 	DefaultProjectID = 1
-
-	localPostgresURL = "postgres://postgres:postgres@localhost:5432/determined?sslmode=disable"
```

Comment:
this is a revert of these changes. i just did `git checkout upstream/main -- master/internal/db/postgres_test_utils.go`